projects/hugo/build.sh

```diff
@@ -0,0 +1,31 @@
+#/bin/bash -eu
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+function compile_fuzzer {
```

Comment:
Can you please remove this and use the new compile_go_fuzzer function, this CL wont work with coverage without this, please see https://github.com/google/oss-fuzz/commit/217175212ba1d484017c32fe6518f85c1aa1273f